### PR TITLE
[Docrevamp][Buildtabs][1684145]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Revamp of build tabs module.

### CODE REVIEW

File: msteams-platform/includes/tabs/dotnet-ngrok-intro.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Establish a secure tunnel to your tab
 
-Microsoft Teams is an entirely cloud-based product and requires that your tab content be available from the cloud using HTTPS endpoints. Teams doesn't allow local hosting. You'll need to either publish your tab to a public URL, or use a proxy that will expose your local port to an internet-facing URL.
+Microsoft Teams is a cloud-based product and requires that your tab content be available from the cloud using HTTPS endpoints. Teams does not allow local hosting. You must either publish your tab to a public URL, or use a proxy that exposes your local port to an internet-facing URL.
 
-To test your tab you'll use [ngrok](https://ngrok.com/docs). Your server's web endpoints will be available while ngrok is running on your local machine. If you close ngrok, the URLs will be different the next time you start it.
\ No newline at end of file
+To test your tab, use [ngrok](https://ngrok.com/docs). Your server's web endpoints are available while ngrok is running on your computer. If you close ngrok, the URLs are different the next time you start it.
```
Comment (lokimeyburg) [inline msteams-platform/includes/tabs/dotnet-ngrok-intro.md:5]: Do we want to mention that ngrok urls only change in the free version of ngrok?

File: msteams-platform/tabs/how-to/build-adaptive-card-tabs.md [hunk 2]
```diff
@@ -6,36 +6,38 @@ ms.topic: conceptual
...
 
-Use Adaptive Cards to build tabs with ease. You can build your tabs with ready-made UI Lego-blocks that look and feel native on desktop, web, and mobile. Building tabs with Adaptive Cards centralizes all Teams app capabilities around a bot backend and Adaptive Card frontend, thus, eliminating the need for a different backend for your bot and tabs. This greatly reduces server and maintenance costs of your Teams app. This article helps you understand the changes required to be made to the app manifest, how the invoke activity requests and sends information in tab with Adaptive Cards, and the impact on the task module workflow. 
+When developing a tab using the traditional method, you must consider these issues, such as HTML and Cascading Style Sheets (CSS) considerations, slow load times, iFrame constraints, and server maintenance and costs. Adaptive Card tabs is a new way to build tabs in Teams. Instead of embedding web content in an IFrame, you can render Adaptive Card to a tab. While the front-end is rendered as Adaptive Card, the backend is powered by a bot. The bot is responsible for accepting requests and responding appropriately with the Adaptive Card that is rendered.
```
Comment (lokimeyburg) [inline msteams-platform/tabs/how-to/build-adaptive-card-tabs.md:15]: 1. This sentence reads unintuitively. Consider something like:
```
When developing a tab using the traditional method, you may run into these issues: ...."
```
2. No one writes out CSS as "Cascading Style Sheets". Can we just say "CSS"?
3. "You can render Adaptive Card**s** to a tab"
4. "While the front-end is rendered with Adaptive Card**s**"

File: msteams-platform/tabs/how-to/build-adaptive-card-tabs.md [hunk 3]
```diff
@@ -6,36 +6,38 @@ ms.topic: conceptual
...
+
+Use Adaptive Cards to build tabs with ease. You can build your tabs with ready-made user interface (UI) Lego-blocks that look and feel native on desktop, web, and mobile. Building tabs with Adaptive Cards centralizes all Teams app capabilities around a bot backend and Adaptive Card frontend. This eliminates the need for a different backend for your bot and tabs. It greatly reduces server and maintenance costs of your Teams app. This article helps you understand the changes required to be made to the app manifest, how the invoke activity requests and sends information in tab with Adaptive Cards, and the impact on the task module workflow.
```
Comment (lokimeyburg) [inline msteams-platform/tabs/how-to/build-adaptive-card-tabs.md:17]: Is this not restating what was said in the previous paragraph?

File: msteams-platform/tabs/how-to/build-adaptive-card-tabs.md [hunk 4]
```diff
@@ -262,7 +264,7 @@ The following code snippets are examples of `task/fetch` request and response:
 
 ### Invoke `task/submit`
 
-The following code snippets are examples of `task/submit` request and response:
+The following code provide examples of `task/submit` request and response:
```
Comment (lokimeyburg) [inline msteams-platform/tabs/how-to/build-adaptive-card-tabs.md:267]: provide**s**

File: msteams-platform/tabs/how-to/conversational-tabs.md [hunk 5]
```diff
@@ -39,31 +40,33 @@ Most of these values can also be retrieved from the `getContext` API.
...
+
+The following image shows the conversation panel with the appropriate conversation:
 
-![Conversationl Sub Entities - Continue Conversation](~/assets/images/tabs/conversational-subentities/continue-conversation.png)
+![Conversational sub-entities - continue conversation](~/assets/images/tabs/conversational-subentities/continue-conversation.png)
 
 ## Enhance a conversation
 
-Finally, it’s important that your tab consumes [deeplinks to your sub-entity](~/concepts/build-and-test/deep-links.md). For example, user clicking the tab chiclet deeplink from the channel conversation. The expected behavior is for you to receive the deeplink, open that sub-entity, and then open the conversation panel for that specific sub-entity.
+Tt is important that your tab includes [deeplinks to your sub-entity](~/concepts/build-and-test/deep-links.md). For example, user selecting the tab chiclet deeplink from the channel conversation. The expected behavior is to receive the deeplink, open that sub-entity, and then open the conversation panel for that sub-entity.
```
Comment (lokimeyburg) [inline msteams-platform/tabs/how-to/conversational-tabs.md:67]: It is

File: msteams-platform/tabs/how-to/conversational-tabs.md [hunk 6]
```diff
@@ -39,31 +40,33 @@ Most of these values can also be retrieved from the `getContext` API.
...
 
-To support conversational sub-entities from your personal or static tab, you do not have to change anything about your implementation. We only support starting or continuing conversations from channel tabs that are already pinned. Supporting static tabs allow you to provide a single location for your users to interact with all your sub-entities. It is, however, important that you save the `subEntityId`, `entityId`, and `channelId` when your tab is originally created in a channel in order for you to have the right properties when opening the conversation view in a static tab.
+To support conversational sub-entities from your personal or static tab, you do not have to change anything in your implementation. Starting or continuing conversations from channel tabs that are already pinned, are only supported. Supporting static tabs allows you to provide a single location for your users to interact with all your sub-entities. It is important that you save the `subEntityId`, `entityId`, and `channelId` when your tab is originally created in a channel to have the right properties when opening the conversation view in a static tab.
```
Comment (lokimeyburg) [inline msteams-platform/tabs/how-to/conversational-tabs.md:69]: Consider active tense:
``We only support starting or continuing conversations from channel tabs...``

File: msteams-platform/tabs/how-to/create-tab-pages/content-page.md [hunk 7]
```diff
@@ -45,69 +48,71 @@ For your page to display in Teams, you must include the [Microsoft Teams JavaScr
...
 
 > [!NOTE]
-> * The behavior on mobile clients is not configurable through this manifest property. Mobile clients show a native loading indicator by default across content pages and iframe-based task modules. This indicator on mobile is shown when a request is made to fetch content and gets dismissed as soon as the request gets completed.
-> * If you indicate  `"showLoadingIndicator : true`  in your app manifest, then all tab configuration, content, and removal pages and all iframe-based task modules must follow the mandatory protocol, below:
+> * The behavior on mobile clients is not configurable through the native loading indicator property. Mobile clients show this indicator by default across content pages and iframe-based task modules. This indicator on mobile is shown when a request is made to fetch content and gets dismissed as soon as the request gets completed.
 
-**To show the loading indicator**
+If you indicate `showLoadingIndicator : true`  in your app manifest, then all tab configuration, content, and removal pages and all iframe-based task modules must follow the mandatory protocol.
```
Comment (lokimeyburg) [inline msteams-platform/tabs/how-to/create-tab-pages/content-page.md:81]: "must follow the mandatory protocol" doesn't sound natural. Consider saying "must follow the protocol/steps below", or something like that.

File: msteams-platform/tabs/how-to/create-tab-pages/removal-page.md [hunk 8]
```diff
@@ -77,11 +76,30 @@ Below is a sample tab removal code block:
 
 ```
 
-When a user selects **Remove** from the tab's drop-down menu, Teams will load the optional `removeUrl` page (designated in your **configuration page**) into an IFrame. Here, the user is presented with a button loaded with the `onClick()` function that calls `microsoftTeams.settings.setValidityState(true)` and enables the **Remove** button located near the bottom of the removal page IFrame.
+When a user selects **Remove** from the tab's drop-down menu, Teams loads the optional `removeUrl` page assigned in your **configuration page**, into an IFrame. The user is shown a button loaded with the `onClick()` function that calls `microsoftTeams.settings.setValidityState(true)` and enables the **Remove** button shown at the bottom of the removal page IFrame.
 
-Following the execution of the remove handler, `removeEvent.notifySuccess()` or `removeEvent.notifyFailure()` notifies Teams of the content removal outcome.
+After the remove handler is executed, `removeEvent.notifySuccess()` or `removeEvent.notifyFailure()` notify Teams of the content removal outcome.
```
Comment (lokimeyburg) [inline msteams-platform/tabs/how-to/create-tab-pages/removal-page.md:81]: notify Teams? consider saying "...notifies Teams" instead.

File: msteams-platform/tabs/what-are-tabs.md [hunk 9]
```diff
@@ -1,86 +1,91 @@
...
+
+You can build tabs with Adaptive Cards and centralize all Teams app capabilities by eliminating the need for a different backend for your bots and tabs. Stage View is a new UI component that allows you to render the content opened in full screen in Teams and pinned as a tab. The existing link unfurling service is updated so that it is used to turn URLs into a tab using an Adaptive Card and Chat Services. You can create conversational tabs using conversational sub-entities that allow users to have conversations about sub-entities in your tab, such as specific task, patient, and sales opportunity, instead of discussing the entire tab.
```
Comment (lokimeyburg) [inline msteams-platform/tabs/what-are-tabs.md:29]: Consider adding links to the documentation of the features you're discussing in this paragraph.

**Review by lokimeyburg — APPROVED** [review]

**Review by Nikitha-MSFT — APPROVED** [review]
Reviewed and approved

File: msteams-platform/tabs/how-to/create-personal-tab.md [hunk 10]
```diff
@@ -0,0 +1,692 @@
...
+author: laujan
+description: A quickstart guide to creating a personal tab with the Yeoman Generator for Microsoft Teams.
+localization_priority: Normal
+ms.topic: quickstart
+ms.author: lajanuar
+---
+
+# Create a personal tab
+
+## Reorder static personal tabs
```
Comment (Wajeed-msft) [inline msteams-platform/tabs/how-to/create-personal-tab.md:12]: I feel reordering can be moved after "Create a custom personal tab".

File: msteams-platform/tabs/what-are-tabs.md [hunk 11]
```diff
@@ -1,77 +1,72 @@
...
 ---
+
 # Microsoft Teams tabs
 
 Tabs are Teams-aware webpages embedded in Microsoft Teams. They are simple HTML <iframe\> tags that point to domains declared in the app manifest and can be added as part of a channel inside a team, group chat, or personal app for an individual user. You can include custom tabs with your app to embed your own web content in Teams or add Teams-specific functionality to your web content. For more information, see [Teams JavaScript client SDK](/javascript/api/overview/msteams-client).
 
-There are two types of tabs available in Teams — channel/group and personal. Channel/group tabs deliver content to channels and group chats, and are a great way to create collaborative spaces around dedicated web-based content. Personal tabs, along with personally-scoped bots, are part of personal apps and are scoped to a single user. They can be pinned to the left navigation bar for easy access.
+The following image shows Contoso channel tabs:
+
+![Channel or group and personal tabs](../assets/images/tabs/tabs.png)
```
Comment (Wajeed-msft) [inline msteams-platform/tabs/what-are-tabs.md:16]: Can we add two images, one for personal tab and another for channel tab?

File: msteams-platform/tabs/what-are-tabs.md [hunk 12]
```diff
@@ -1,77 +1,72 @@
...
+
+There are few prerequisites that you must go through before working on tabs.
+
```
Comment (Wajeed-msft) [inline msteams-platform/tabs/what-are-tabs.md:19]: Can we move this sentence after the video? currently it feels like videos are about prerequesites.

File: msteams-platform/tabs/how-to/tab-requirements.md [hunk 13]
```diff
@@ -8,32 +8,47 @@ ms.topic: conceptual
...
-* For authentication to work on mobile clients, you must upgrade you Teams JavaScript SDK to at least version 1.4.1.
+* For authentication to work on mobile clients, you must upgrade Teams JavaScript SDK to at least version 1.4.1.
 
 * If you choose to have your channel or group tab appear on Teams mobile clients, the `setSettings()` configuration must have a value for the `websiteUrl` property.
 
 * MS Teams tab does not support the ability to load intranet websites that use self-signed certificates.
 
+## See also
+
+* [Teams tabs](~/tabs/what-are-tabs.md)
```
Comment (Wajeed-msft) [inline msteams-platform/tabs/how-to/tab-requirements.md:39]: I feel there are too many see also links. Do we have any fixed guidelines on number of links?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-abirade — APPROVED** [review]
Approval

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**